pkg/k8shandler/status.go

```diff
@@ -49,3 +54,35 @@ func updateNodeStatus(node *nodeState, dpl *v1alpha1.ElasticsearchStatus) {
 	}
 	dpl.Nodes = append(dpl.Nodes, nodeStatus)
 }
+
+func clusterHealth() string {
+	config := &ExecConfig{
+		namespace:      "openshift-logging",
```

Comment:
We should abstract this to a constant.  Ideally if we could reuse the operator to be more generic this is something that would be probably passed into the operator

Reply:
We shouldn't be hardcoding this, it should be available from `dpl.Namespace`